LION/experiments/ct_benchmarking_experiments.py

```diff
@@ -142,12 +108,11 @@ def default_parameters(dataset="2DeteCT"):
         param.data_loader_params.task = "sino2recon"
         param.data_loader_params.input_mode = "mode2"
         param.data_loader_params.target_mode = "mode2"
-        
+
         return param
 
 
 class LimitedAngle90CTRecon(Experiment):
-
     def __init__(self, experiment_params=None, dataset="2DeteCT", datafolder=None):
         super().__init__(experiment_params, dataset, datafolder)
 
```

Comment:
Below there should then also be 
param.name = (
            "90 degrees Limited Angle CT reconstruction experiment from mode 2"
        )

Reply:
It is! Github may have hiden you unchanged code, but its there 5 lines below.

Reply:
I know that the basic content is there. But you changed the style of the code with brackets and new lines, which should be the same for all experiments, shouldn't it?

Reply:
@mbkiss thats the `black` autoformater, we let it do what it does 🤷